### ISSUE TITLE
Step Size not provided

### ISSUE THREAD

**Issue by Debjyotinath** [Ornolfr/RatingView#2]
Hi,
Very good stuff, it really helps my need.
But the only problem is when my is_indicator is false i can increase the rating by one star only, and not even by half star.

**Comment by Ornolfr** [comment]
@Debjyotinath there is not any possibility for user to set half star rating, only from 1 to `max_count` obviously. And I don't get the problem clearly. Do you mean when you interact with the view you can't change rating from 1 star to 4 for example?

**Comment by Debjyotinath** [comment]
Yes, when i interact with the rating i can increment my rating +1 i.e 1 to 2 to 3 and so on. But i cannot increment by 0.5 i.e 1 to 1.5 to 2 to 2.5. However a half star drawable is there, so my guess was i should be able to increment by 0.5.

**Comment by Ornolfr** [comment]
@Debjyotinath  The main thing is this line:
`setRating(Math.round(event.getX() / getWidth() * mMaxCount + 0.5));`
There cannot be any limitations with already starred drawables. It just checks the place where you clicked, compares it with view's width and sets the rating dependent on `max_count`. The only problem there might be if you set padding to the view, it's not supported due to realization difficulty.

**Comment by Debjyotinath** [comment]
I downloaded your project and executed it without any change.I was not able to do it in that.

**Comment by murali0286** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @idoru, @Leii, @benemohamed, @brandonvanha, @quqiuzhu, @mikebuss, @objcguy, @kazunobu-fujii, @scaca, @dwqs, @wanjj8023, @kakaxihoracker, @karanthukral, @bleeckerj, @mman, @ivanzotov, @Napoleon-D, @hh778, @qwzyes, @brentleyjones

**Comment by dwqs** [comment]
您的邮件已经收到，我会尽快回复，谢谢！
